example_test.go

```diff
@@ -75,17 +75,18 @@ func Example_basicConfiguration() {
 	// See the documentation for Config and zapcore.EncoderConfig for all the
 	// available options.
 	rawJSON := []byte(`{
-		"level": "debug",
-		"encoding": "json",
-		"outputPaths": ["stdout", "/tmp/logs"],
-		"errorOutputPaths": ["stderr"],
-		"initialFields": {"foo": "bar"},
-		"encoderConfig": {
-			"messageKey": "message",
-			"levelKey": "level",
-			"levelEncoder": "lowercase"
-		}
+	  "level": "debug",
+	  "encoding": "json",
+	  "outputPaths": ["stdout", "/tmp/logs"],
+	  "errorOutputPaths": ["stderr"],
+	  "initialFields": {"foo": "bar"},
+	  "encoderConfig": {
+	    "messageKey": "message",
+	    "levelKey": "level",
+	    "levelEncoder": "lowercase"
+	  }
 	}`)
```

Comment:
Thanks for the explanation, I was a little confused initially -- then figured it was just the contents in the backticks that now use spaces. As long as `gofmt` is happy, looks good.